components/p3a/constellation_log_store.cc

```diff
@@ -223,4 +223,9 @@ void ConstellationLogStore::LoadPersistedUnsentLogs() {
   }
 }
 
+const metrics::LogMetadata ConstellationLogStore::staged_log_metadata() const {
+  DCHECK(has_staged_log());
```

Comment:
i wouldn't even bother with `DCHECK``s

Reply:
Fixed in 0830baa69e7566a88bd1961e72f76acf68493ff1.